external/llvm-project/mlir/lib/Conversion/GPUToROCDL/LowerGpuOpsToROCDLOps.cpp

```diff
@@ -200,6 +200,10 @@ struct LowerGpuOpsToROCDLOpsPass
         op->setAttr(ROCDL::ROCDLDialect::getFlatWorkGroupSizeAttrName(),
                     flatSizeAttr);
       }
+      if (auto minWavesPerEu = op->removeAttr("min_wave_per_eu")
```

Comment:
I don't think, in this case, that we need this, and we should instead set `rocdl.waves_per_eu` directly